src/TSID/src/SE3Task.cpp

```diff
@@ -260,8 +261,9 @@ bool SE3Task::update()
     m_R3Controller.computeControlLaw();
 
     // get the output
-    bFullDoF.head<3>() += getControllerOutupt(m_R3Controller);
-    bFullDoF.tail<3>() += getControllerOutupt(m_SO3Controller);
+    m_controllerOutput.head<3>() = getControllerOutupt(m_R3Controller);
+    m_controllerOutput.tail<3>() = getControllerOutupt(m_SO3Controller);
```

Comment:
I just realized that the lambda ``getControllerOutupt`` has a typo, and if we fix it, it has the same name of the method you added. Could you call the lambda ``getControllerOutupt`` in a different way?

Reply:
fyi @Gio-DS

Reply:
I'm going to apply this modification suggested by @S-Dafarra and I merge the PR

Reply:
Done in https://github.com/ami-iit/bipedal-locomotion-framework/pull/740/commits/de84e1f820c28f192bc9d3d7958fb2bffbd97f84